### PR TITLE
feat: 🎸 Filters - widget

### PR DESCRIPTION
**Pull request by maciejrybaniec**

### CODE REVIEW

File: src/components/FilterSettings/FilterSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+    detachedWidgetConnections,
+    eventStream,
+    targetProperty,
+    eventStreamsPool,
+    eventStreamSchema,
+    schemaProcessing,
+  } = useSelector(getFilterSettings);
+  const chartWidgetsCount = useSelector(getCurrentDashboardChartsCount);
+
+  const [isEditMode] = useState(!!targetProperty);
```
Comment (apryka) [inline src/components/FilterSettings/FilterSettings.tsx:60]: do we need `useState` for this?

File: src/components/FilterManagement/FilterManagement.tsx [hunk 2]
```diff
@@ -29,15 +25,17 @@ type Props = {
   isHoverActive: boolean;
   /** Remove widget event handler */
   onRemoveWidget: () => void;
+  /** Edit widget event handler */
+  onEditWidget: () => void;
 };
 
-const DatePickerManagement: FC<Props> = ({
-  id,
+// TODO: Handle labels
```
Comment (apryka) [inline src/components/FilterManagement/FilterManagement.tsx:32]: please remove if its not necessary

File: src/components/FilterSettings/FilterSettings.tsx (hunk 1 above)
Comment (dariuszlacheta) [reply src/components/FilterSettings/FilterSettings.tsx:60]: +1

Comment (maciejrybaniec) [reply src/components/FilterSettings/FilterSettings.tsx:60]: Yes, as `targetProperty` could change in store and the button label should stay the same. It is used to show `create` or `edit` label.

**Review by dariuszlacheta — APPROVED** [review]

**Review by apryka — APPROVED** [review]

**Pull request merged**